KsApi/models/UpdateBackingEnvelopeTests.swift

```diff
@@ -0,0 +1,31 @@
+@testable import KsApi
+import XCTest
+
+final class UpdateBackingEnvelopeTests: XCTestCase {
+  func testUpdateBackingEnvelopeDecoding() {
+    let jsonString = """
+    {
+      "updateBacking": {
+        "checkout": {
+          "state": "VERIFYING",
+          "backing": {
+            "requiresAction": false,
+            "clientSecret": "super-secret"
+          }
+        }
+      }
+    }
+    """
+
+    let data = Data(jsonString.utf8)
+
+    do {
+      let envelope = try JSONDecoder().decode(UpdateBackingEnvelope.self, from: data)
+      XCTAssertEqual(envelope.updateBacking.checkout.state, .verifying)
+      XCTAssertEqual(envelope.updateBacking.checkout.backing.requiresAction, false)
+      XCTAssertEqual(envelope.updateBacking.checkout.backing.clientSecret, "super-secret")
```

Comment:
Since `clientSecret` can be `null` I'm not sure if you want to maybe add another test to make sure it decodes properly in that scenario.